src/Compilers/Core/Portable/Hashing/XxHash128.cs

```diff
@@ -117,7 +117,7 @@ public static bool TryHash(ReadOnlySpan<byte> source, Span<byte> destination, ou
             return false;
         }
 
-#if NET7_0_OR_GREATER
+#if NET8_0_OR_GREATER
```

Comment:
i don't understaand this change.   it wraps using Uint128.  Which was added in .net 7.

Reply:
From https://github.com/dotnet/roslyn/issues/75453#issuecomment-2405500584:

> When people see `#if NET7` in code but no `net7.0` in the TFM list then there is a tendency to ask "wait, is that stale? can we remove that?" Hence I like to roll them forward to make sure that we still need them and to make it clearer to readers that they are intended.

Reply:
so in this case, we want to move to NET8 pragma, even though it's really that it was NET7 that added this api?  if that's the design, then that's fine with me. I just wanted to confirm that was how we think it's best to do things.